### PR TITLE
860 ic top navigation double reading

### PR DESCRIPTION
**Pull request by GCHQ-Developer-299**
## Summary of the changes
Added a silent prop to the ic-tooltip, for instances where the tooltip is duplicating the visible text of the child component. 

Set the tooltips for icon-variant buttons and navigation-items to be silent, as they already have accessible names. 

I've tested this thoroughly with Voiceover (both by tabbing through and moving the virtual cursor) on icon buttons, side/top navigation and ic-tooltip. I've also added a new "silent" story to the ic-tooltip. 

I am unsure how this gels with the [mozilla guidance on role=tooltip](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) since this change ignores it. Functionally this does solve the double-reading bug and preserves functionality of the tooltip though.

## Related issue
#860

## Checklist
- [ ] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] All acceptance criteria reviewed and met. 
- [ ] Accessibility Insights FastPass performed.
- [ ] A11y unit test added and yields no issues.
- [ ] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [ ] Page can be zoomed to 400% with no loss of content. 
- [ ] Screen magnifier used with no issues. 
- [ ] Text resized to 200% with no loss of content.
- [ ] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.
- [ ] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [ ] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [ ] Min/max content examples tested with no loss of content or overflow. 
- [x] Browser support tested (Chrome, Safari, Firefox and Edge).
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 
- [ ] All prop combinations work without issue.

### CODE REVIEW

File: packages/web-components/src/components/ic-tooltip/ic-tooltip.stories.mdx [hunk 1]
```diff
@@ -31,6 +31,43 @@ import readme from "./readme.md";
   </Story>
 </Canvas>
 
+### Silent
+
+<Canvas>
+  <Story name="Silent" height="100px">
+    {(args) =>
+      html`<ic-tooltip
```
Comment (jd239) [inline packages/web-components/src/components/ic-tooltip/ic-tooltip.stories.mdx:39]: Hovering over this button produces two tooltips. One generated from the browser and the other, an `ic-tooltip`

File: packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx [hunk 2]
```diff
@@ -271,11 +242,11 @@ export class Tooltip {
   render() {
     const { label } = this;
     return (
-      <Host class={{ "ic-tooltip": true }}>
+      <Host class={{ "ic-tooltip": true }} title={!this.silent ? label : null}>
```
Comment (jd239) [inline packages/web-components/src/components/ic-tooltip/ic-tooltip.tsx:245]: Title will produce the browser tooltip as well as the `ic-tooltip`

**Review by jd239 — CHANGES_REQUESTED** [review]
Wondering if another way to look at this ticket is to figure out whether aria-hidden on the tooltip should be set to true or false or would that prevent any of it's children such as button from being read out?

**Comment by GCHQ-Developer-299** [comment]
PR has become stale and there's significant feedback to address. Will close and return with another branch.

**Pull request closed without merging**